src/dotnet/commands/dotnet-tool/install/LocalizableStrings.resx

```diff
@@ -195,16 +202,25 @@ Tool '{1}' (version '{2}') was successfully installed.</value>
   <data name="InvalidNuGetVersionRange" xml:space="preserve">
     <value>Specified version '{0}' is not a valid NuGet version range.</value>
   </data>
-  <data name="InstallToolCommandNeedGlobalOrToolPath" xml:space="preserve">
-    <value>Please specify either the global option (--global) or the tool path option (--tool-path).</value>
-  </data>
-  <data name="InstallToolCommandInvalidGlobalAndToolPath" xml:space="preserve">
-    <value>(--global) conflicts with the tool path option (--tool-path). Please specify only one of the options.</value>
+  <data name="InstallToolCommandInvalidGlobalAndLocalAndToolPath" xml:space="preserve">
+    <value>Cannot have options: {0}. The local option(--local), the global option (--global), the tool path option (--tool-path), can only have one at a time. Please specify only one of the options.</value>
   </data>
   <data name="ToolPathOptionName" xml:space="preserve">
     <value>PATH</value>
   </data>
   <data name="ToolPathOptionDescription" xml:space="preserve">
     <value>The directory where the tool will be installed. The directory will be created if it does not exist.</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="ManifestPathOptionDescription" xml:space="preserve">
+    <value>Path to the manifest file.</value>
+  </data>
+  <data name="ManifestPathOptionName" xml:space="preserve">
+    <value>PATH</value>
```

Comment:
I am not sure we localize environment variable names.

Reply:
It has always been done this way. Like this https://github.com/dotnet/cli/blob/master/src/dotnet/commands/dotnet-build/LocalizableStrings.resx#L135

I haven't thought about. And I don't have an opinion. It seems never been translated. However, it also make sense if we translate it.

Reply:
Yes, we should not localize environment variable names, just like we don't localize APIs.

Reply:
This isn't an environment variable name.  This is what is displayed as the option name in help:

Example:

```
--tool-path <PATH>        The directory where the tool will be installed. The directory will be created if it does not exist.
```